tests/dummy/app/pods/components/frost-create-form-with-scroll/template.hbs

```diff
@@ -1,18 +1,22 @@
-{{#frost-modal-input title='Test title' subtitle='Subtitle'}}
-{{!#frost-modal-input titleComponent='myTitle'}}
-
-  {{#perfect-scroll
+{{#frost-modal-input
+  title='Test title'
+  subtitle='Subtitle'
+  openButton='Open Modal with Perfect Scroll'
```

Comment:
I'd prefer to emphasis this format

```handlebars
{{#ember-remodal as |modal|}}
    {{#modal.open}}
      {{!-- Anything you put here will open the modal on click and fire the 'onOpen' action --}}
    {{/modal.open}}
  {{/ember-remodal}}
```

Instead of the `openButton` in the attrs